packages/mml-web/src/MMLClickTrigger.ts

```diff
@@ -54,6 +61,71 @@ export class MMLClickTrigger {
     }
   }
 
+  private handleTouchEnd(event: TouchEvent) {
+    if (Date.now() - touchTimestamp < touchThresholdMilliseconds) {
+      /* a short touch, i.e., a click */
+      if ((event.detail as any).element) {
+        // Avoid infinite loop of handling click events that originated from this trigger
+        return;
+      }
+      let x = 0;
+      let y = 0;
+      if (!document.pointerLockElement) {
+        let offsetX = touchX;
+        let offsetY = touchY;
+        let width = window.innerWidth;
+        let height = window.innerHeight;
+        if (this.clickTarget instanceof HTMLElement) {
+          width = this.clickTarget.offsetWidth;
+          height = this.clickTarget.offsetHeight;
+        }
+        if (event.target) {
+          /* get the equivalent of event.offset in a mouse event */
+          const bcr = (event.target as HTMLElement).getBoundingClientRect();
+          offsetX = offsetX - bcr.x;
+          offsetY = offsetY - bcr.y;
+        }
+        x = (offsetX / width) * 2 - 1;
+        y = -((offsetY / height) * 2 - 1);
+      }
+      this.raycaster.setFromCamera(new THREE.Vector2(x, y), this.scene.getCamera());
```

Comment:
From this point in this method this is duplicated from `handleClick`. It's likely best to extract this into a private method to avoid them diverging unintentionally.